### DISCUSSION TITLE
Is there a reason why `window` is defined when testing an api route?

### DISCUSSION THREAD

**Discussion by seanonthenet** [Xunnamius/next-test-api-route-handler#770] [General]
The `window` object seems to be defined when running an API test. 

Is this expected?

**Comment by Xunnamius** [comment]
Are you using Jest with the `jsdom` test environment?

**Reply by seanonthenet** [reply]
Oh yeah... I have the `jest.config.js` configured as per [this example](https://github.com/vercel/next.js/blob/canary/examples/with-jest/jest.config.js).

Is there a recommended config for NTARH?

**Reply by Xunnamius** [reply]
It seems that `window` is defined is because of the Jest environment you're using. If you don't want `window` to be defined, you'll have to choose a different Jest environment or setup something custom. Either way, `window` being defined or not defined should have no bearing on the functionality of NTARH.
